pkg/connector/users.go

```
@@ -74,12 +81,17 @@ func (b *userBuilder) Grants(ctx context.Context, userResource *v2.Resource, _ *
 	} else {
 		// All the Job Permissions of the user will be requested in order to create a grant for any role
 		// for which the user has at least one Job with it.
-		userJobPermissions, err := b.client.GetJobPermissionsOfAUser(ctx, user.ID)
+		userJobPermissions, rateLimitData, err := b.client.GetJobPermissionsOfAUser(ctx, &tokens, user.ID)
```

the `err` should always be checked before doing anything with the other arguments. if an err might happen we should use errors.Is to handle a well known error but we can't ignore the error

---

Here is quite the same! The error is handled just after adding the rate limit data to the annotation, but I see that it may be a misunderstanding from my side.
Thanks! I will modify it